Improve tablet breakpoints on map page

lucascumsille opened the pull request:
Fixes: https://github.com/mysociety/societyworks/issues/3784
Replaces: https://github.com/mysociety/fixmystreet/pull/4566/

# FMS
## /around
https://github.com/user-attachments/assets/d9e14f46-c752-49ef-a5b9-d44a6a58c25e


## /report/556
https://github.com/user-attachments/assets/f89eb283-21f9-4a5d-b900-f06572528a45


## /admin/report_edit/101
https://github.com/user-attachments/assets/c0b074b3-393f-486a-bfc7-33841ab96773

# Highways
## /around

https://github.com/user-attachments/assets/8863268d-c8b0-46d3-a949-e6bbaba9810f

## /admin/report_edit/101

https://github.com/user-attachments/assets/fd934dae-6e05-45ef-8d82-a472cc50e41f



[skip changelog]

lucascumsille commented on web/cobrands/sass/_layout.scss:
```diff
@@ -328,28 +426,67 @@ body.mappage.admin {
   // with the edge of .shadow-wrap, we give #side-report a fixed width,
   // and leave the other column to shrink to the remaining space.
   #side-report {
-    width: $mappage-sidebar-width;
```
Let's go with consistency. I added the missing line.

lucascumsille commented:
Thanks for the feedback @dracos =)
Here is a list of the things I did with the commit fixups:

>Zurich doesn't display correctly, I'm afraid, I think that needs to remain at 27em for all possibilities, given its header and navigation (see pic), and I think its notes can always remain at 15em, they don't need to get wider.
I added a fixup for this.

> It seems odd to now have 20em sidebar width at 60em
I set it up at 25em now

> Was there a reason the defaults changed a bit for the normal view, e.g. a report page logged in as admin, currently at large width is 29/25, but in this PR is 28/22 at large or 30/23 at xlarge.

At the beginning I thought have it wider didn't make much sense with the examples I had because there wasn't that much information. But I just increased the size so it's wider considering all thereports with longer info and a narrow side bar will cause more vertical scrolling.

dracos reviewed (changes requested):
One suggestion just to tweak the smallest (non-mobile) sidebar width a bit.
I can look at the broken JS tests, which are due to the fact they have to click on X,Y coordinate of the map, which has now moved...

lucascumsille commented:
@dracos thanks for all your help with this =). I have rebased to master and squashed the fixup commits. I added your commit to fix the test errors to this commit: [Improved responsiveness for #map_sidebar](https://github.com/mysociety/fixmystreet/pull/5592/commits/5a9c1239bea6a35204369e0a0aab66ff05669373)
Let me know if you are happy with having these 4 commits.

dracos reviewed (changes requested):
I tidied up the commit messages, but sorry, while doing that I've just spotted possibly one more thing - on an All Reports body page, the "Assign to..." dropdown menu (which is the only other place that uses govuk-fieldset-wrapper) now has a full width dropdown and the "Assign" button has wrapped to a new line, which looks okay I guess, but wondered if you'd realised/wanted to change that at all.

Before:

<img width="443" height="309" alt="Image" src="https://github.com/user-attachments/assets/fbf00012-f667-42ca-a31d-ef8e30b3ca2e" />

After:

<img width="472" height="361" alt="Image" src="https://github.com/user-attachments/assets/46b3cb28-065b-48a3-a516-62b33bb7e7fa" />

Perhaps it should not be using govuk-fieldset-wrapper, or should be using it in a more consistent way with the other usage?

lucascumsille commented:
> I tidied up the commit messages, but sorry, while doing that I've just spotted possibly one more thing - on an All Reports body page, the "Assign to..." dropdown menu (which is the only other place that uses govuk-fieldset-wrapper) now has a full width dropdown and the "Assign" button has wrapped to a new line, which looks okay I guess, but wondered if you'd realised/wanted to change that at all.
> 
> Before:
> 
> <img alt="Image" width="443" height="309" src="https://private-user-images.githubusercontent.com/154364/482488584-fbf00012-f667-42ca-a31d-ef8e30b3ca2e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTYyNzQwNTMsIm5iZiI6MTc1NjI3Mzc1MywicGF0aCI6Ii8xNTQzNjQvNDgyNDg4NTg0LWZiZjAwMDEyLWY2NjctNDJjYS1hMzFkLWVmOGUzMGIzY2EyZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyN1QwNTQ5MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNmI0YmI4NWVkNzRiN2MwMTQ0MThkMjEwYWFjNDgxZTNhOWI5ZjY5YmYxOTdkNmM2Njc4Mzg4MDMwMTYzZDU4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.7vnxRz449flujhjy5MF30bixoKbDGUeLeDHbPpaEFwU">
> After:
> 
> <img alt="Image" width="472" height="361" src="https://private-user-images.githubusercontent.com/154364/482488630-46b3cb28-065b-48a3-a516-62b33bb7e7fa.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTYyNzQwNTMsIm5iZiI6MTc1NjI3Mzc1MywicGF0aCI6Ii8xNTQzNjQvNDgyNDg4NjMwLTQ2YjNjYjI4LTA2NWItNDhhMy1hNTE2LTYyYjMzYmI3ZTdmYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyN1QwNTQ5MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kZTBjODZkZDI4NjU2ZTc2YzNmODMxMzViZDkyZmMyZDE3NmYxNjZiMjUzYTYxMDZiYjU1YTUzYzhjMmY0ZDRjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.mlBs5Ncfhm0q_-x5gLDfBU_5PMAILjWQYNS1zcvJf00">
> Perhaps it should not be using govuk-fieldset-wrapper, or should be using it in a more consistent way with the other usage?

I gave them the same treatment as the select items above. So I wrapped the select and button with a `filter-group` div so it aligns with the rest. Let me know what you think and if you'd like me to create a new class with a better naming technically these two are not filters, but I guess they are in way.

<img width="473" height="528" alt="Screenshot 2025-08-27 at 07 22 36" src="https://github.com/user-attachments/assets/f0955a59-1171-4049-a3ad-8595a244dad8" />
